pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -774,7 +774,7 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
     @FieldContext(
         category = CATEGORY_POLICIES,
         doc = "How often is the thread pool scheduled to check whether a snapshot needs to be taken."
-                + "(disable with value 0)"
+                + "(must be greater than 0)"
```

Comment:
I am considering whether we should move this change to another thread. Do we have any cases that need to disable the current snapshot?

Reply:
Created a separate PR https://github.com/apache/pulsar/pull/22480 to do this change